LCLS locks openliberty.properties preventing `mvn clean` when server.xml open in editor

So I wanted to at least raise this for awareness...not sure what can easily be done...

But if I have my **server.xml** file open in Liberty Tools Eclipse and need to do a clean `mvn clean` (FWIW, Liberty Tools Eclipse doesn't offer a clean function but that's kind of besides the point here), it sometimes fails because the file is locked.

I assume this is locked by the LCLS lemminx ext:

> [INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ batch-bonuspayout-application ---
> [INFO] Deleting C:\git\sample.batch.bonuspayout\target
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  2.240 s
> [INFO] Finished at: 2023-10-04T14:46:28-04:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal org.apache.maven.plugins:maven-clean-plugin:2.5:clean (default-clean) on project batch-bonuspayout-application: Failed to clean project: Failed to delete C:\git\sample.batch.bonuspayout\target\liberty\wlp\lib\versions\openliberty.properties -> [Help 1]

And...the workaround is just to try again until it works, which is a pretty obvious a thing to try as well. so not a huge deal.   Just noting.

--- a/lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/data/LibertyRuntime.java
+++ b/lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/data/LibertyRuntime.java
@@ -29,8 +29,7 @@ public class LibertyRuntime {
      */
     public LibertyRuntime(Path propertiesFilePath) {
 
-        try {
-            FileInputStream fis = new FileInputStream(propertiesFilePath.toFile());
+        try (FileInputStream fis = new FileInputStream(propertiesFilePath.toFile())) {
             Properties prop = new Properties();
             prop.load(fis);
 

